proposals/001-kroxylicious-operator-api-v1alpha.md

```diff
@@ -231,20 +231,8 @@ spec:
          x: y
 
   # Points to the cluster being proxied.  Can either be Strimzi Kafka resource or endpoint details.
-  targetCluster:
-    clusterRef:
-      group: strimzi.io|kroxylicious.io # must be strimzi.io or kroxylicious.io
-      kind: Kafka|KafkaClusterRef  # must be Kafka (belonging to group strimzi.io) or TargetClusterRef (belonging to group kroxylicious.io)
-      name: my-cluster
-      listenerName: listener # name of strimzi listener, permitted only for Kafka (group strimzi.io).  (Optional)
-    tls:
-      # Optional - client auth certificate
-      # secret provided by the Developer.
-      certificateRef:
-        kind: Secret # if present must be Secret, otherwise defaulted to Secret
-        group: ""  # if present must be "", otherwise defaulted to ""
-        name: clientcert
-        namespace: # namespace of the secret, if omitted assumes namespace of this resource
+  kafkaServiceRef: # required
```

Comment:
I think it can help people understand the resource if we include the role that the referred-to thing plays in the name of the property. I.e. `targetKafkaServiceRef` says a lot more to someone unfamiliar with the API than `kafkaServiceRef`.

Reply:
done